libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizer.cs

```diff
@@ -45,9 +45,9 @@ public class LuisRecognizer : IRecognizer
         /// <param name="clientHandler">Custom handler for LUIS API calls.</param>
         public LuisRecognizer(LuisApplication application, LuisPredictionOptions predictionOptions = null, bool includeApiResults = false, HttpClientHandler clientHandler = null)
```

Comment:
"LUIS _application" on line 42 has an extra space in it.

Reply:
Line 42. Add a <see> tag to LuisApplication. 
Line 43. Add a <see> tag to LuisPredictionOptions.
Line 43. Add (Optional) to the start of the text. 
Line 44. Add (Optional) to includeApiResults
Line 45. Add (Optional) to the clientHandler.

Reply:
Done.